generators/typescript/express/generator/src/contexts/type-schema/TypeSchemaContextImpl.ts

```diff
@@ -83,14 +83,16 @@ export class TypeSchemaContextImpl implements TypeSchemaContext {
 
     public getGeneratedTypeSchema(typeName: DeclaredTypeName): GeneratedTypeSchema {
         const typeDeclaration = this.typeResolver.getTypeDeclarationFromName(typeName);
+        const examples = typeDeclaration.autogeneratedExamples;
+        examples.push(...typeDeclaration.userProvidedExamples);
```

Comment:
I think we only want to only use the user-provided examples if they're available. If not, that's when we use the auto-generated ones.